qml/js/functions.js

```diff
@@ -47,9 +47,9 @@ function getMessageText(message, simple, currentUserId, ignoreEntities) {
             return enhanceMessageText(message.content.text, ignoreEntities);
         }
     case 'messageSticker':
-        return simple ? qsTr("Sticker: %1").arg(message.content.sticker.emoji) : "";
+        return message.content.sticker.emoji
     case 'messageAnimatedEmoji':
-        return simple ? qsTr("Animated Emoji: %1").arg(message.content.animated_emoji.sticker.emoji) : "";
+        return message.content.animated_emoji.sticker.emoji
```

Comment:
This change causes sticker emojis and animated emojis to duplicate in chats. You should still check for the `simple` parameter.

Reply:
Right! Good catch.

Reply:
Fixed!